environments/templates/rubintv-application.yaml

```diff
@@ -0,0 +1,37 @@
+{{- if .Values.rubintv.enabled -}}
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: "rubintv"
+spec:
+  finalizers:
+    - "kubernetes"
```

Comment:
Delete this; I cleaned this up already for all the other applications, since it isn't needed.